web/admin.py

```
@@ -623,6 +624,38 @@ class LearningStreakAdmin(admin.ModelAdmin):
+        ("Dates", {
+            "fields": ("created_at", "verified_at", "expires_at", "used_at")
+        }),
+    )
+    
+    def mark_as_verified(self, request, queryset):
+        for discount in queryset.filter(status="pending"):
+            discount.mark_as_verified()
+        self.message_user(request, f"{queryset.filter(status='pending').count()} discounts marked as verified.")
+    mark_as_verified.short_description = "Mark selected discounts as verified"
+    
+    def mark_as_expired(self, request, queryset):
```

Ruff (0.8.2)

Missing return type annotation for public function `mark_as_expired`

Add return type annotation: `None`

(ANN201)